app/api/routes/markets.py

```diff
@@ -0,0 +1,29 @@
+import logging
+from typing import Any
+from fastapi import APIRouter, Depends, HTTPException
+from sqlalchemy.orm import Session
+from app.api.dependencies import database
+from app.models.domain import markets
+from app.crud import crud_markets, crud_posts
+
+router = APIRouter()
+
+
+@router.post("/create")
+def create_market_posts(
+    *, db: Session = Depends(database.get_db), market_in: markets.MarketCreate
+) -> Any:
+    """
+    Create new user.
```

Comment:
Annotation mistake

Reply:
Do you mean 'Any'? Any type can cast any type. We didn't discuss about response type so I used Any type